components/molecules/DevCard/dev-card.tsx

```diff
@@ -184,8 +184,8 @@ export default function DevCard(props: DevCardProps) {
                   <div className="text-xs">PRs created</div>
                 </div>
                 <div className="text-center">
-                  <div className="text-6xl font-black">{props.isLoading ? "-" : props.contributions}</div>
-                  <div className="text-xs">Contributions</div>
+                  <div className="text-6xl font-black">{props.isLoading ? "-" : props.repos}</div>
+                  <div className="text-xs">{props.repos! <= 1 ? "Repo" : "Repos"}</div>
```

Comment:
This needs to be exact
```suggestion
                  <div className="text-xs">{props.repos! === 1 ? "Repo" : "Repos"}</div>
```

Reply:
Very well.

But, this does mean that in cases where the `repos` value is `0`, the term takes its plural form i.e. "0 Repos"

Is that fine?